src/app/app.component.ts

```diff
@@ -7,5 +8,34 @@ import { Component } from '@angular/core';
     standalone: false
 })
 export class AppComponent {
-  title = 'cv';
-}
+    isSidenavOpen = false;
+    activeTabIndex = 0; 
+    darkTheme: boolean = false;
```

Comment:
boolean type is not required.
don't add types for: string, number, boolean. Typescript do it for you under the hood